scripts/cpu-setonix.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash --login
+
+# SLURM directives
+#
+# Here we specify to SLURM we want an OpenMP job with 32 threads
```

Comment:
For the sake of avoiding confusion, we should probably get rid of the reference to "an OpenMP job with 32 threads", since Quokka doesn't use OpenMP.